docs/index.asciidoc

```diff
@@ -217,13 +217,15 @@ The endpoint should be an HTTP or HTTPS URL, e.g. https://example.com
   * Value type is <<string,string>>
   * Default value is `""`
 
-Specify a prefix to the uploaded filename, this can simulate directories on S3.
+Specify a prefix to the uploaded filename to simulate directories on S3.
 Prefix does not require leading slash.
 This option supports
 {logstash-ref}/event-dependent-configuration.html#sprintf[Logstash
-interpolation]; for example, files can be prefixed with the event date using
+interpolation]. For example, files can be prefixed with the event date using
 `prefix = "%{+YYYY}/%{+MM}/%{+dd}"`.
-Be warned this can create a lot of temporary local files.
+
+IMPORTANT: Prefixing files can create many temporary local files, and can
```

Comment:
I was thinking of something along the lines of: 

"Take care when using complex/high cardinality interpolation with prefixes, particularly when combined with a `rotation_strategy` that would delay uploads, as this may cause an excessive numbers of in-progress uploads which may cause performance and stability issues"